src/main/config/secor.common.properties

```diff
@@ -46,6 +46,30 @@ aws.secret.key=
 aws.region=
 aws.endpoint=
 
+###########################
+# START AWS S3 ENCRYPTION #
+###########################
+
+# AWS specify type of server-side encryption, if any
+# set to S3 to enable S3-managed encryption
+# set to KMS to enable AWS KMS-managed encryption (see aws.sse.kms.key)
+# set to customer to enable customer-managed encryption (see aws.sse.customer.key)
+# set empty to disable encryption
+aws.sse.type=
```

Comment:
If you don't set the value for this new property, does it throw exception when you call mConfig.getProperty on this new one?